internal/db/queries.sql

```diff
@@ -34,7 +34,7 @@ WITH dequeued AS (
 	WHERE id IN (
 		SELECT id FROM mergestat.repo_sync_queue
 		WHERE status = 'QUEUED'
-		ORDER BY repo_sync_queue.created_at ASC, repo_sync_queue.id ASC LIMIT 1 FOR UPDATE SKIP LOCKED
+		ORDER BY repo_sync_queue.priority_level ASC,repo_sync_queue.created_at ASC, repo_sync_queue.id ASC LIMIT 1 FOR UPDATE SKIP LOCKED
```

Comment:
```suggestion
		ORDER BY repo_sync_queue.priority_level DESC, repo_sync_queue.created_at ASC, repo_sync_queue.id ASC LIMIT 1 FOR UPDATE SKIP LOCKED
```

we'll want higher priorities to be dequeued first (I assume), so I think we want this DESC

Reply:
Ascending seems right to me priority 1 goes before priority 2.

Reply:
You're right - that makes more sense to me too